packages/react-components/src/components/VideoGallery.tsx

```diff
@@ -710,6 +710,8 @@ export const VideoGallery = (props: VideoGalleryProps): JSX.Element => {
       onDisposeRemoteStreamView={onDisposeRemoteScreenShareStreamView}
       isReceiving={screenShareParticipant.screenShareStream?.isReceiving}
       participantVideoScalingMode={selectedScalingModeState[screenShareParticipant.userId]}
+      /* @conditional-compile-remove(ppt-live) */
```

Comment:
add same TODO here
`// TODO: need to remove after contentSharingRole avaible in WebCalling SDK.`